src/components/UI/DownloadButton/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import protectedAxios from 'utils/protectedAxios';
+import { AxiosResponse } from 'axios';
+import Loading from 'components/UI/Loading';
+import FetchError from 'components/UI/FetchError';
+
+type DownloadButtonProps = {
+  downloadLink: string;
+};
+const DownloadButton: React.FC<DownloadButtonProps> = ({ downloadLink }) => {
+  const [loading, setLoading] = React.useState<boolean>(false);
+  const [downloadError, setDownloadError] = React.useState<any | null>(null);
+  const downloadFile = () => {
+    setLoading(true);
+    protectedAxios
+      .get<Blob>(downloadLink, {
+        responseType: 'blob',
+      })
+      .then((response: AxiosResponse<Blob>) => {
+        const blob = new Blob([response.data], {
+          type: response.headers['content-type'],
+        });
+        const url = window.URL.createObjectURL(blob);
+        const a: HTMLAnchorElement = document.createElement('a');
+        a.href = url;
+        const filename = downloadLink.split('/').pop();
+        a.download = filename;
+        const body: any = document.getElementsByTagName('body')[0];
+        body.appendChild(a);
+        a.click();
+        body.removeChild(a);
```

Comment:
Is this the only way to trigger the download? You can't use a javascript navigation like `window.open` ?
Maybe this is the point of the future [FileSystem web API](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemWritableFileStream/write#browser_compatibility) but that isn't on Safari yet...

Reply:
If the add-click-remove link element *is* the only way to do this, perhaps we could add this data-download link underneath the already-clicked link element. That way if some browser security feature prevents the js click event for whatever reason, a user could manually click it?

Reply:
Great. Yeh. Let's merge this and give it a try on some big studies on staging; and hopefully then release it to unblock existing private-study users 👍 If you can add a JIRA task for the followup exploration too, that would be great. Thanks!

Reply:
I've created the JIra task as requested